libs/barista-components/confirmation-dialog/src/confirmation-dialog.spec.ts

```diff
@@ -146,10 +141,28 @@ describe('ConfirmationDialogComponent', () => {
       tick(DT_CONFIRMATION_POP_DURATION);
       fixture.componentInstance.testState = null;
       fixture.detectChanges();
-      tick(DT_CONFIRMATION_FADE_DURATION);
+      tick();
       expect((DtConfirmationDialog as any)._activeDialog).toBeNull();
       flush();
     }));
+
+    it('should wiggle', fakeAsync(() => {
+      let componentInstance: TestComponent = fixture.componentInstance;
+      expect(componentInstance.dialog._wiggleState).toEqual(false);
+      componentInstance.dialog.wiggle();
+      expect(componentInstance.dialog._wiggleState).toEqual(true);
+      fixture.detectChanges();
+      tick();
+      expect(componentInstance.dialog._wiggleState).toEqual(false);
+      fixture.detectChanges();
+
+      // wiggle again
+      componentInstance.dialog.wiggle();
+      expect(componentInstance.dialog._wiggleState).toEqual(true);
+      fixture.detectChanges();
+      tick();
+      expect(componentInstance.dialog._wiggleState).toEqual(false);
+    }));
   });
 
```

Comment:
A testcase where we check whether a change in the viewport resizer changes the left position would be nice.